src/amazonqGumby/activation.ts

```diff
@@ -32,7 +32,7 @@ export async function activate(context: ExtContext) {
             if (transformByQState.isRunning()) {
                 confirmStopTransformByQ(transformByQState.getJobId())
             } else {
-                vscode.window.showInformationMessage(CodeWhispererConstants.noOngoingJobMessage)
+                vscode.window.showInformationMessage(CodeWhispererConstants.noOngoingJobMessage, { modal: true })
```

Comment:
I've noticed a lot of use of "modal" in gumby workflow. They should be avoided. Why does this one in particular need to be modal?

Reply:
It doesn't, and can be changed back. We felt that the modal fit in better, and I wasn't aware that modals should be avoided -- is there a particular reason for that?

Reply:
Looking at this again, it actually probably makes more sense to _not_make this a modal, so good point. But still curious on why modals should be avoided - I guess because they can be disruptive?

Reply:
> still curious on why modals should be avoided - I guess because they can be disruptive?

Yes. There is a ton of discussion to be found on this via web search, but the short answer is that they demand a user decision. Things that don't actually require a user decision should not demand a user decision.